v2/fleet-local/control/marathon@.service

```diff
@@ -21,10 +21,13 @@ ExecStartPre=-/usr/bin/docker rm marathon
 ExecStart=/usr/bin/sh -c "/usr/bin/docker run \
```

Comment:
you're probably going to need something more along the lines of:

`Environment="USER=etcdctl get /marathon/username"`

And in the command below **for the container** `docker run ... $($IMAGE) --http_credentials $($USER):$($PW)`